Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaTranslation.swift

```diff
@@ -279,7 +281,11 @@ extension JNISwift2JavaGenerator {
         genericRequirements: functionSignature.genericRequirements
       )
 
-      let resultType = try translate(swiftResult: functionSignature.result)
+      var resultType = try translate(swiftResult: functionSignature.result)
+
+      if functionSignature.effectSpecifiers.contains(.async) {
```

Comment:
Maybe let's add isAsync to SwiftFunctionSignature as well

```suggestion
      if functionSignature.isAsync {
```